po/LINGUAS

```diff
@@ -0,0 +1,127 @@
+# please keep this list sorted alphabetically
+#
+af
```

Comment:
Should be without locale. If a language is translated at transifex ( pull it with `tx pull -af`) you will see a *.po file in that directory. Then you can add the locale name to LINGUAS.
Here you mark which language translations (*.po files) are available for this project.